avail/distro/src/avail/Avail.avail/Foundation.avail/Foundation.avail

```diff
@@ -1042,6 +1042,11 @@ Extends
 		"character set lexer",
 		"style lexer"
 	),
+	"Libraries" =
+	(
+		"Link_",
+		"Link pojos_"
```

Comment:
We can talk about this as a group, but I would strongly prefer the very explicit name: `Link library:_`